app/__main__.py

```diff
@@ -226,10 +226,10 @@ class Config(app_commands.Group):
     async def enable(self, interaction: Interaction) -> None:
         """Enable the game on the current channel."""
         if interaction.channel.id in await interaction.client.database.get_channels(interaction.guild.id):
-            await interaction.response.send_message("The game is already enabled on this channel")
+            await interaction.response.send_message("The game is already enabled on this channel", ephemeral=True)
         else:
             await interaction.client.database.enable_channel(interaction.guild.id, interaction.channel.id)
-            await interaction.response.send_message("Enabled the game on this channel")
+            await interaction.response.send_message("Enabled the game on this channel", ephemeral=True)
```

Comment:
Sorry, this can be non-ephemeral too. It's not a game channel at that point yet.